### PR TITLE
Add support for labels with `pr:` prefix

### PR DESCRIPTION
**Pull request by vegaro**
We are going to add `pr:` as a prefix for Pull Request labels. This PR adds support for both versions, while we transition all repos to use `pr:` labels

### CODE REVIEW

File: lib/fastlane/plugin/revenuecat_internal/helper/github_helper.rb [hunk 1]
```diff
@@ -8,7 +8,9 @@ module Fastlane
 
   module Helper
     class GitHubHelper
-      SUPPORTED_PR_LABELS = %w[breaking build ci docs feat fix perf revenuecatui refactor style test next_release dependencies phc_dependencies minor].to_set
+      PR_LABEL_PREFIX = 'pr:'
+      BASE_LABELS = %w[breaking build ci docs feat fix perf revenuecatui refactor style test next_release dependencies phc_dependencies minor]
+      SUPPORTED_PR_LABELS = (BASE_LABELS + BASE_LABELS.map { |label| "#{PR_LABEL_PREFIX}#{label}" }).to_set
```
Comment (tonidero) [inline lib/fastlane/plugin/revenuecat_internal/helper/github_helper.rb:13]: Hmm I've been thinking whether it makes sense to just remove support for the old labels now, and just update each SDK to the new version when the labels have been updated... Wdyt?

Comment (vegaro) [reply lib/fastlane/plugin/revenuecat_internal/helper/github_helper.rb:13]: Right. Good point. I'm over complicating things. With the dangerfile we can't do that, but we can for the fastlane plugin

File: lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb [hunk 2]
```diff
@@ -264,17 +264,17 @@ def self.append_phc_version_if_necessary(append_phc_version_if_next_version_is_n
 
       # rubocop:disable Metrics/PerceivedComplexity
       private_class_method def self.get_section_depending_on_types_of_change(change_types)
-        if change_types.include?("breaking")
+        if change_types.any?("pr:breaking")
```
Comment (tonidero) [inline lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb:267]: Hmm why the change from `include?` to `any?`

**Review by tonidero — APPROVED** [review]
Just a question but LGTM otherwise

File: lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb (hunk 2 above)
Comment (vegaro) [reply lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb:267]: oopsies, this is from before, where it could be with or without prefix. Good catch

**Review by tonidero — APPROVED** [review]
